rclcpp/include/rclcpp/subscription.hpp

```diff
@@ -159,12 +160,26 @@ class Subscription : public SubscriptionBase
     return message_memory_strategy_->borrow_message();
   }
 
-  std::shared_ptr<rcl_serialized_message_t> create_serialized_message()
+  std::shared_ptr<rcl_serialized_message_t> create_serialized_message() override
   {
     return message_memory_strategy_->borrow_serialized_message();
   }
 
-  void handle_message(std::shared_ptr<void> & message, const rmw_message_info_t & message_info)
+  void *
+  loan_message() override
+  {
+    CallbackMessageT * msg = nullptr;
+    return msg;
+  }
+
+  rclcpp::LoanedMessageSequence
+  loan_message_sequence() override
```

Comment:
What are these methods used for?

Reply:
I guess they are more for consistency when implementing the executor (in the style of `sub->create_message()`, `sub->handle_message()`, `sub->return_message()`). But I am fine with only having the `return_loaned_message()` functions.